privacyidea/lib/tokens/pushtoken.py

```diff
@@ -335,6 +335,12 @@ def get_class_info(key=None, ret='all'):
                            'type': 'int',
                            'desc': _("The user may only have this maximum number of active Push tokens assigned."),
                            'group': GROUP.TOKEN
+                       },
+                       'push_' + ACTION.FORCE_APP_PIN: {
+                           'type': 'bool',
+                           'group': "PUSH",
+                           'desc': _('Enforce setting an app pin for the privacyIDEA '
```

Comment:
I think this is a bit misleading since it is not a PIN for the whole app, but just for the token. I would word it like 'Enforce requesting the Smartphone PIN/Biometrics before Push requests can be accepted'

Reply:
@nilsbehlen You are right. However, this has changed also for HOTP and TOTP...

@plettich ...I am not sure, if we want to change this *now*?

Reply:
@cornelinux @nilsbehlen I'll change it for the Push-Token, but for HOTP/TOTP we'll keep it that way for now and change it in v3.8.

Reply:
Done in 5a7f9dd5